e2e/tests/cohorts/cohorts-rename.spec.ts

```diff
@@ -43,24 +43,27 @@ describe('User can create, modify, rename and delete Cohort', () => {
     const minAge = 21;
     const maxAge = 95;
     const group1 = cohortBuildPage.findIncludeParticipantsGroup('Group 1');
-    const group1Count = await group1.includeAge(minAge, maxAge);
```

Comment:
In staging, there is a enabled UI feature flag which is disabled in test.
UI behaviors are different as result. I had to modify this test so it works in both env.

Reply:
I believe it was `enableCBAgeTypeOptions`. But I did a quick search now, flag was removed 3 days ago for all env. So I don't know what happens next.

What the feature flag did wass: the count on the age slider is pre-calculated and then the real count is calculated in bigquery when Finish button in Modal is clicked. Pre-calcuated count is different from calculated count.

Reply:
would `expect(totalCount).toEqual(group1Count)` not then fail with the feature flag enabled?

Reply:
No, test shouldn't fail with or without flag.
Old way was to get the count within Age modal then verify count after close Age modal. New way is get the calculated Group Count after close Age modal, and then compare it with the Total Count. Because only one group, Group Count and Total Count should match.

Reply:
Gotcha, thanks.